From a6992b15421a37aeb56ac7fee65733596e6870e4 Mon Sep 17 00:00:00 2001
From: Jean Trivelly <jean.trivelly@intel.com>
Date: Fri, 1 Aug 2014 16:17:02 +0200
Subject: [PATCH] Handle custom wpa_supplicant.conf management

BZ: 214330

Adress custom wpa_supplicant.conf management since it may be
handled from other locations.

Change-Id: Id8c38239974dab04dc790df0e0d831dd405c54dc
Category: device enablement
Domain: CWS.WIFI-Common
Origin: internal
Upstream-Candidate: yes
Signed-off-by: Jean Trivelly <jean.trivelly@intel.com>
Signed-off-by: Cedric Vamour <cedric.vamour@intel.com>
---
 bcmdhd/config/Android.mk |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/bcmdhd/config/Android.mk b/bcmdhd/config/Android.mk
index b125897..5fa3d6d 100644
--- a/bcmdhd/config/Android.mk
+++ b/bcmdhd/config/Android.mk
@@ -27,6 +27,7 @@ include $(BUILD_PREBUILT)
 #########################
 
 WIFI_DRIVER_SOCKET_IFACE := wlan0
+ifneq ($(strip $(CUSTOM_WPA_SUPPLICANT_CONF)),true)
 ifeq ($(strip $(WPA_SUPPLICANT_VERSION)),VER_0_8_X)
   include external/wpa_supplicant_8/wpa_supplicant/wpa_supplicant_conf.mk
 else
@@ -36,4 +37,5 @@ else
   include external/wpa_supplicant/wpa_supplicant_conf.mk
 endif
 endif
+endif
 #######################
-- 
1.7.9.5

